### PR TITLE
Update set-up-a-web-experiment.md

### PR DESCRIPTION
**Pull request by akhil-prakash**

### CODE REVIEW

File: content/collections/web_experiment/en/set-up-a-web-experiment.md [hunk 1]
```diff
@@ -50,8 +50,10 @@ If the script isn’t present on the page you specified, Amplitude Experiment ca
 
 ![web-exp-4.png](/docs/output/img/workflow/web-exp-4.png)
 
+You can also exclude the experiment from running on certain pages. 
+
 12. Next, target the users you want to include in this experiment. If you're familiar with feature experiment targeting, Web Experiment [audience targeting](/docs/web-experiment/targeting#audience-targeting) works differently.
-13. The *Advanced* tab provides several [additional options](/docs/feature-experiment/workflow/finalize-statistical-preferences) for your experiment. Note that several of the options discussed in that article are not available for Web Experiments.
+13. The *Advanced* tab provides several [additional options](/docs/feature-experiment/workflow/finalize-statistical-preferences) for your experiment.
```
Comment (akhil-prakash) [inline content/collections/web_experiment/en/set-up-a-web-experiment.md:56]: Everything here https://amplitude.com/docs/feature-experiment/workflow/finalize-statistical-preferences is supported in both web and feature experiment

File: content/collections/web_experiment/en/set-up-a-web-experiment.md [hunk 2]
```diff
@@ -50,8 +50,10 @@ If the script isn’t present on the page you specified, Amplitude Experiment ca
 
 ![web-exp-4.png](/docs/output/img/workflow/web-exp-4.png)
 
+You can also exclude the experiment from running on certain pages.
```
Comment (akhil-prakash) [inline content/collections/web_experiment/en/set-up-a-web-experiment.md:53]: Feel free to reword this to make it nicer

File: content/collections/web_experiment/en/set-up-a-web-experiment.md (hunk 1 above)
Comment (markzegarelli) [reply content/collections/web_experiment/en/set-up-a-web-experiment.md:56]: Good callout. I'll find a way to single-source this content in both locations.

**Review by markzegarelli — APPROVED** [review]

File: content/collections/web_experiment/en/set-up-a-web-experiment.md (hunk 1 above)
Comment (akhil-prakash) [reply content/collections/web_experiment/en/set-up-a-web-experiment.md:56]: To clarify what I meant, I meant that this line ` Note that several of the options discussed in that article are not available for Web Experiments.` is not correct. There are articles under "Feature Experiment" that apply to both "Feature Experiment" and "Web Experiment". https://amplitude.atlassian.net/wiki/spaces/EXP/pages/2684518402/Web+Experiment+Documentation We decided to put web specific stuff under "Web Experiment" but if it was for both Feature and Web Experiment it goes under "Feature Experiment"
